code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -225,8 +226,13 @@
 	else
 		pump_unload(M)
 	bolt_open = !bolt_open
+	icon_state = "moistnugget_open"
+	update_icon()
```

Comment:
```suggestion
	update_icon(UPDATE_ICON_STATE)
```
This is all being handled in `update_icon_state()` now, so you don't need to manually do it here, and you need to tell the code what part of `update_icon()` you want to run.